addons/actions.md

```diff
@@ -168,6 +168,111 @@ end
+- `isInDayset("<set>", <datetime>)`: returns `true` if the day defined by the passed in Joda DateTime `<datetime>` os in dayset `<set>`; returns `false` otherwise.
+- `isWeekend`: returns `true` if today is a weekend, `false` otherwise.
+- `isWeekend(<offset>)`: returns true if the day `<offset>` days from today is a weekend. Use a negative value to check days in the past.
+- `isWeekend(<datetime>)`: returns `true` if the day defined by the passed in Joda DateTime is a weekend, `false` otherwise.
+
+#### Scripted Automation
+
+One must import the Ephemeris Action and then call the above functions using `Ephemeris.<function>`, for example `Ephemeris.getNextBankHoliday()`.
+
+### Configuration
+
+#### PaperUI
+
+In PaperUI one has the ability to configure which days of the week are considered weekends and set your country, region, and city.
+If no country is provided the service uses your system default locale settings.
+Browse to Configuration and System and scroll down to the Ephemeris section.
+Setting the country, region and optionally city will cause Ephemeris to use the list of bank holidays defined for that location by [Jollyday](https://github.com/svendiedrichsen/jollyday).
+You can find the XML file for your country [here](https://github.com/svendiedrichsen/jollyday/tree/master/src/main/resources/holidays) and the localized mapping files that map between the holiday name in the XML and your preferred language are located [here](https://github.com/svendiedrichsen/jollyday/tree/master/src/main/resources/descriptions).
+
+When filling in the country, region and city, if there isn’t a drop down list for one after choosing the one(s) higher up (e.g. there is no drop down list for city after choosing country and region), leave that field blank.
+There are no special bank holidays defined by Jollyday for that level.
+
+#### `ephemeris.cfg`
+
+By default Ephemeris supports a weekend dayset and a school dayset.
+You can define additional daysets or modify the school dayset.
+If you need to define custom daysets, you must do all your configuration through the ephemeris.cfg file instead of using PaperUI as described above so make sure to at least redefine the weekend dayset.
+If there is no `dayset-weekend` defined, calls to `isWeekend` will generate errors.
+
+Config file location: `$OH_CONF/services/ephemeris.cfg`.
+
+Fields:
+- `country`: country to use to get the built in list of bank holidays, uses the standard two letter country code.
+- `region`: uses the appropriate two letter region code.
+- `dayset-<name>`: list of the days of that are in this custom dayset.
+
+If in doubt on what values to use, see the links above to find the XML file for your country, open the XML file, and find the tns:SubConfigurations for your region and use the “hierarchy” value for region.
+
+For example:
+
+```
+country=de
+region=nw
+dayset-workday=Monday,Tuesday,Wednesday,Thursday,Friday
+dayset-weekend=Saturday,Sunday
+dayset-trash=Monday
```

Comment:
Sry, I missed that in the community thread. We should use the new list configuration feature. And uppercase is preferred for Ephemeris configuration. Otherwise the UIs may struggle.

```
dayset-workday=[MONDAY,TUESDAY,WEDNESDAY,THURSDAY,FRIDAY]
dayset-weekend=[SATURDAY,SUNDAY]
dayset-trash=[MONDAY]
```

Reply:
Will be corrected in the upcoming commit. Double check the text I added to the description for dayset-name to verify that I didn't make a false assumption (i.e. the names of the days of the week must be in English as well as all caps).

Reply:
:+1: